### ISSUE TITLE
Wrong 60px icon in iOS

### ISSUE THREAD

**Issue by felixhammerl** [MobileChromeApps/mobile-chrome-apps#502]
Hi guys,

for iOS, icon-60.png is not only the wrong file, it is also 196*196. Here's my manifest:
[general manifest](https://github.com/whiteout-io/mail-html5/blob/master/src/manifest.json#L7-L10)
[mobile delta](https://github.com/whiteout-io/mail-html5/blob/master/src/manifest.mobile.json#L10-L27)

Am I doing something wrong here? Seems weird that this bug only exists for the 60px icon, the rest seems fine.

I am using the cca built from the master, but this bug was also present in previous versions:
```
> cca --version
0.5.2-dev
> git show HEAD
commit 9b8fa2da6e5057417cc6757e63953f5e43c8daa5
Author: Andrew Grieve <agrieve@chromium.org>
Date:   Tue Jan 6 10:41:15 2015 -0500

    Add GET_ACCOUNTS permission to chrome.identiy (fixes #495)
```

If I can help to track this down, just let me know.

Cheers
Felix

**Comment by mmocny** [comment]
(More context in #399.)

**Comment by jpchase** [comment]
@agrieve is this issue also addressed by commit 81c555e73fc5f96bb1486149c4e8ab5f0745a75d?

**Comment by agrieve** [comment]
Likely enough that we should close this and assume so :)
